docs/pipeline-components-and-applications/legacy/snowflake-loader/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Snowflake Loader"
+title: "Snowflake Loader (legacy)"
```

Comment:
Do we need (legacy) in the title if its in the legacy section with `nofollow`?

Reply:
Yes (I think you mean `noindex` rather than `nofollow`). For the on-site search I’ve configured Algolia to still index the `noindex` pages but rank them lower (in case people want to search for terms that are only present on the deprecated pages, e.g. “shredder”). However I noticed that “Snowflake Loader” still comes up at the top of the search for Snowflake, simply because there is nothing else with “Snowflake” in the title! So I am both adding “Snowflake” to the title of RDB loader and marking this one as legacy in case it still comes up below it.

Reply:
Yeah, I did mean `noindex` 🤦 

Sounds good 👍